hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/staking/StakingUtilities.java

```diff
@@ -53,7 +54,7 @@ public static boolean hasStakeMetaChanges(
         }
         final var differDeclineReward = originalAccount.declineReward() != modifiedAccount.declineReward();
         final var differStakedNodeId =
-                !originalAccount.stakedNodeIdOrElse(0L).equals(modifiedAccount.stakedNodeIdOrElse(0L));
+                !originalAccount.stakedNodeIdOrElse(-1L).equals(modifiedAccount.stakedNodeIdOrElse(-1L));
```

Comment:
Could use `SENTINEL_NODE_ID` here for improved readability.

Reply:
Changed. Thanks